SimplenoteUITests/SimplenoteUITestsSearch.swift

```diff
@@ -1,22 +1,43 @@
+    tags: ["sea-monster", "reptile", "prehistoric"]
+)
+
+let kingGong = NoteData(
+    name: "King Kong",
+    content: "King Kong is a film monster, resembling an enormous gorilla, that has appeared in various media since 1933. He has been dubbed The Eighth Wonder of the World, a phrase commonly used within the films. The character first appeared in the novelization of the 1933 film King Kong from RKO Pictures, with the film premiering a little over two months later. The film received universal acclaim upon its initial release and re-releases. A sequel quickly followed that same year with The Son of Kong, featuring Little Kong. In the 1960s, Toho produced King Kong vs. Godzilla (1962), pitting a larger Kong against Toho's own Godzilla, and King Kong Escapes (1967), based on The King Kong Show (1966–1969) from Rankin/Bass Productions. In 1976, Dino De Laurentiis produced a modern remake of the original film directed by John Guillermin.",
+    tags: ["ape", "prehistoric"]
+)
+
+let mechagodzilla = NoteData(
+    name: "Mechagodzilla",
+    content: "Mechagodzilla (メカゴジラ, Mekagojira) is a fictional mecha character that first appeared in the 1974 film Godzilla vs. Mechagodzilla. In its debut appearance, Mechagodzilla is depicted as an extraterrestrial villain that confronts Godzilla. In subsequent iterations, Mechagodzilla is usually depicted as a man-made weapon designed to defend Japan from Godzilla. In all incarnations, the character is portrayed as a robotic doppelgänger with a vast array of weaponry, and along with King Ghidorah, is commonly considered to be an archenemy of Godzilla.",
+    tags: ["man-made", "robot"]
+)
+
+let diacritic = NoteData(
+    name: "Diacritic",
+    content: "A diacritic (also diacritical mark, diacritical point, diacritical sign, or accent) is a glyph added to a letter or basic glyph. The term derives from the Ancient Greek διακριτικός (diakritikós, \"distinguishing\"), from διακρίνω (diakrī́nō, \"to distinguish\"). Some diacritical marks, such as the acute ( ´ ) and grave ( ` ), are often called accents. Examples are the diaereses in the borrowed French words naïve and Noël, which show that the vowel with the diaeresis mark is pronounced separately from the preceding vowel; the acute and grave accents, which can indicate that a final vowel is to be pronounced, as in saké and poetic breathèd; and the cedilla under the \"c\" in the borrowed French word façade, which shows it is pronounced /s/ rather than /k/. In other Latin-script alphabets, they may distinguish between homonyms, such as the French là (\"there\") versus la (\"the\") that are both pronounced /la/. In Gaelic type, a dot over a consonant indicates lenition of the consonant in question.",
+    tags: ["language", "diacritic"]
+)
+
+// swiftlint:enable line_length
+let allNotes: [NoteData] = [
+    godzilla,
+    kingGong,
+    mechagodzilla,
+    diacritic
+]
```

Comment:
Here's one advantage: because we have values distinct values for each note rather than a collection of constants, we can define an array with all the notes. This will simplify the tests that look for all the notes.

Reply:
@mokagio this is precious, thank you, and thank you for the [value type](https://www.objc.io/issues/16-swift/swift-classes-vs-structs/) read!